cluster/node-pools/master-default/userdata.yaml

```diff
@@ -883,6 +883,15 @@ write_files:
           qps: {{ .Cluster.ConfigItems.event_rate_limit_config_qps }}
           burst: {{ .Cluster.ConfigItems.event_rate_limit_config_burst }}
 {{ end }}
+{{- if index .NodePool.ConfigItems "sysctl_settings"}}
+  {{- range $i, $cfgs := split .NodePool.ConfigItems.sysctl_settings ","}}
+  {{- $cfg := split $cfgs "="}}
+  - owner: root:root
+    path: /etc/sysctl.d/0{{$i}}-{{index $cfg 0}}.conf
+    content: |
+      {{index $cfg 0}} = {{index $cfg 1}}
+  {{- end}}
```

Comment:
Why not put them into a single file (i.e. move range into the `content`)?

Reply:
Good idea. Will do.